content/2015/11/15/logo-samples.rst

```diff
@@ -1,48 +0,0 @@
-public: yes
-author: sondra
-tags: ['Case Studies']
-image:
-  - src: 'blog/2016/logos/oddbird.jpg'
-summary: |
-  OddBird offers full brand identity development,
-  including logo design.
-  Here's a sampling of our work.
-
-
-Brand Identity: Logos
-=====================
-
-We've helped develop several brands over the years.
-Here are a few of our favorites...
-
-.. figure:: /static/images/blog/2016/logos/coachhub.jpg
-
-   *CoachHub: responsive wellness coach app for ORCAS*
-
-.. figure:: /static/images/blog/2016/logos/coachhub-avatars.jpg
-
-   *CoachHub Avatars*
-
-.. figure:: /static/images/blog/2016/logos/susy.jpg
-
-   *Susy: open source web layout tools*
-
-.. figure:: /static/images/blog/2016/logos/moztrap.jpg
-
-   *MozTrap: QA management tool for Mozilla*
-
-.. figure:: /static/images/blog/2016/logos/carrotproject.jpg
-
-   *The Carrot Project: Social and environmental impact reviews*
-
-.. figure:: /static/images/blog/2016/logos/oddbooks.jpg
-
-   *OddBooks: Collaborative writing for multimedia artists*
-
-.. figure:: /static/images/blog/2016/logos/teacupgorilla.jpg
-
-   *Teacup Gorilla: indie rock artist website*
-
-.. figure:: /static/images/blog/2016/logos/portfoliyo.jpg
-
-   *Portfoliyo Mascot: responsive education app for K Network*
```

Comment:
I agree with the thrust. Maybe @SondraE wants to make a story for adding it back in a better format?